tests/functional/cylc-poll/06-loadleveler.t

```diff
@@ -35,9 +35,9 @@ set_test_number 2
 
 create_test_global_config "" "
 [platforms]
-    [[loadleveler-platform]]
+    [[${BATCH_SYS_NAME}-test-platform]]
```

Comment:
Tests for Slurm and PBS symlinked to this file so it needs to provide platforms.